pkgs/tools/networking/aria2/default.nix

```diff
@@ -13,10 +12,10 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ openssl c-ares libxml2 sqlite zlib libssh2 ] ++
-    stdenv.lib.optional stdenv.isDarwin Security;
+  buildInputs = [ openssl c-ares libxml2 sqlite zlib libssh2 ];
 
-  configureFlags = [ "--with-ca-bundle=/etc/ssl/certs/ca-certificates.crt" ];
+  configureFlags = [ "--with-ca-bundle=${cacert}/etc/ssl/certs/ca-bundle.crt" ] ++
+    stdenv.lib.optional stdenv.isDarwin [ "--with-openssl" "--without-appletls" ];
```

Comment:
Thanks for doing all this testing. I don't really know what's the best way forward. I mean the best is if we could use Apple's to make the integration seamless. Right now if the user installs another CA in the keychain aria2 won't know about it. It's going to generate support issues where users will have one site working only with aria2 or reversely.